static/app/components/forms/fields/fileField.tsx

```diff
@@ -39,25 +41,22 @@ export default function FileField({accept, ...props}: FileFieldProps) {
   };
 
   return (
-    <InputField
-      {...props}
-      accept={accept?.join(', ')}
-      field={({onChange, ...fieldProps}) => {
-        return (
-          <Fragment>
-            <Input
-              {...omit(fieldProps, 'value', 'onBlur', 'onKeyDown')}
-              type="file"
-              onChange={e => handleFile(onChange, e)}
-            />
-            {isUploading ? (
-              <div>This is a janky upload indicator. Wait to hit save!</div>
-            ) : (
-              ''
-            )}
-          </Fragment>
-        );
-      }}
-    />
+    <FormField {...props}>
+      {({children: _children, onChange, ...fieldProps}) => (
+        <Fragment>
+          <Input
+            {...omit(fieldProps, 'value', 'onBlur', 'onKeyDown')}
+            type="file"
+            accept={accept?.join(', ')}
+            onChange={e => handleFile(onChange, e)}
+          />
+          {isUploading ? (
+            <div>This is a janky upload indicator. Wait to hit save!</div>
```

Comment:
> Janky is a slang term for something run down, of poor quality, or unreliable


TIL.

Reply:
don't we need to translate here? 
```suggestion
            <div>{t("This is a janky upload indicator. Wait to hit save!")}</div>
```

Reply:
lol, I think cramer built this for something internal